test/features/auth/presenters/login_presenter_test.dart

```diff
@@ -1,28 +1,136 @@
+import 'package:flutter_demo/core/domain/model/user.dart';
+import 'package:flutter_demo/features/auth/domain/model/log_in_failure.dart';
 import 'package:flutter_demo/features/auth/login/login_initial_params.dart';
 import 'package:flutter_demo/features/auth/login/login_presentation_model.dart';
 import 'package:flutter_demo/features/auth/login/login_presenter.dart';
+import 'package:flutter_demo/localization/app_localizations_utils.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
 
+import '../../../test_utils/test_utils.dart';
 import '../mocks/auth_mock_definitions.dart';
+import '../mocks/auth_mocks.dart';
 
 void main() {
   late LoginPresentationModel model;
   late LoginPresenter presenter;
   late MockLoginNavigator navigator;
+  late MockLogInUseCase logInUseCase;
+
+  // blocTest<LoginPresenter, LoginViewModel>(
```

Comment:
These two tests fail, but I didn't remove them and just commented because in order for them to pass I just need to extend 'Equatable' in 'LoginPresentationModel'. Initially I did, but while running 'make check', I got an error that I have a 'not allowed' package imported in the 'login_presentation_model.dart' which was the one needed for 'equatable'. With the 'equatable' being extended, the file 'auth_mock_definitions.dart' was also complaining that the classes that override '==' shouldn't be implemented. So I chose to not extend the 'Equatable' in the end but keep these commented tests just to get your opinion on this topic, to see if we should remove this package constraint for 'Equatable' and let the presentation models to be able to have this capability.